04LessonFour/in-class-project/index.html

```diff
@@ -40,27 +40,27 @@
 
         <aside id="aside_computer">
 
-            <img src="img/processor.jpg" </>
+            <img src="img/processor.jpg" /> alt="computer image"/>
 
         </aside>
 
 
-    <section id=_Intel_Customer_Support>
+    <section id="_Intel_Customer_Support">
 
         <h3> Intel Customer Support</h3>
-        <p> Find Support Topics by Product, Drivers & Software, or ask the community <a id=customer_support href="www.http://www.intel.com/content/www/us/en/support/contact-support.html">read more</a>
+        <p> Find Support Topics by Product, Drivers & Software, or ask the community <a id="customer_support" href="www.http://www.intel.com/content/www/us/en/support/contact-support.html">read more</a>
 
 
         </p>
 
 
     </section>
 
```

Comment:
great work! just be careful not to start any class names or IDs with special characters.